app/src/main/java/org/projectbuendia/client/ui/OdkActivityLauncher.java

```diff
@@ -180,6 +201,78 @@ public static void showOdkCollect(
         callingActivity.startActivityForResult(intent, requestCode);
     }
 
+    private static List<OpenMrsXformIndexEntry> getLocalFormEntries() {
+        List<OpenMrsXformIndexEntry> entries = new ArrayList<>();
+
+        final ContentResolver resolver = App.getInstance().getContentResolver();
+        Cursor c = resolver.query(Contracts.Forms.CONTENT_URI, null, null, null, null);
```

Comment:
Thx @capnfabs. I specified the projection here and from the source method `DbSyncHelper#getFormUpdateOps` where i've copied it.